v2/internal/controllers/crd_cache_redis_20230801_test.go

```diff
@@ -114,7 +114,7 @@ func Redis_LinkedServer_20230801_CRUD(tc *testcommon.KubePerTestContext, _ *reso
 	// secondary server.
 	serverRole := cache.RedisLinkedServerCreateProperties_ServerRole_Secondary
 	linkedServer := cache.RedisLinkedServer{
-		ObjectMeta: tc.MakeObjectMetaWithName(redis2.ObjectMeta.Name),
+		ObjectMeta: tc.MakeObjectMetaWithName(redis2.Name),
```

Comment:
Just curious: Do you remember what linter had this suggestion and what exactly it was? They both sorta seem OK to me, so wanting to understand what the issue is

Reply:
I don't remember - our `Taskfile` includes `--fix` so these changes were automatic.